src/EditorFeatures/Core/Implementation/IntelliSense/Helpers.cs

```diff
@@ -72,7 +140,60 @@ internal static IEnumerable<object> BuildClassifiedTextElements(ImmutableArray<T
             return paragraphs;
         }
 
-        internal static object CreateParagraphFromLines(IReadOnlyList<ClassifiedTextElement> lines)
+        private static void NavigateToQuickInfoTarget(string navigationTarget, Document document, IEnumerable<Lazy<IStreamingFindUsagesPresenter>> streamingPresenters)
+        {
+            var navigateToLinkService = document.Project.Solution.Workspace.Services.GetRequiredService<INavigateToLinkService>();
+            if (Uri.TryCreate(navigationTarget, UriKind.Absolute, out var absoluteUri))
+            {
+                navigateToLinkService.TryNavigateToLinkAsync(absoluteUri, CancellationToken.None);
+                return;
+            }
+
+            SymbolKeyResolution resolvedSymbolKey;
+            try
+            {
+                resolvedSymbolKey = SymbolKey.Resolve(navigationTarget, document.Project.GetCompilationAsync(CancellationToken.None).WaitAndGetResult(CancellationToken.None), cancellationToken: CancellationToken.None);
+            }
+            catch
+            {
+                // Ignore symbol resolution failures. It likely is just a badly formed URI.
+                return;
+            }
+
+            if (resolvedSymbolKey.GetAnySymbol() is object)
+            {
+                GoToDefinitionHelpers.TryGoToDefinition(resolvedSymbolKey.Symbol, document.Project, streamingPresenters, CancellationToken.None);
```

Comment:
you use .GetAnySymbol, but then .Symbol.  these may not be the same.    i.e. the latter may be null when the former is not.